assembl/auth/util.py

```diff
@@ -494,12 +505,32 @@ def add_user(name, email, password, role, force=False, username=None,
             created_localrole = True
             db.add(LocalUserRole(
                 user=user, role=localrole, discussion=discussion))
-    # Do this at login
-    # if discussion:
-    #     user.get_notification_subscriptions(discussion.id)
-    if kwargs.get('flush', True):
+
+    if flush:
         db.flush()
 
+    status_in_discussion = None
+    if resend_if_not_logged_in and discussion and not (created_user or created_localrole):
+        status_in_discussion = user.get_status_in_discussion(discussion.id)
+    if send_password_change and (
+            created_user or created_localrole or (
+                resend_if_not_logged_in and (
+                    status_in_discussion is None or
+                    not status_in_discussion.first_visit))):
+        from assembl.views.auth.views import send_change_password_email
+        from assembl.models import Discussion
```

Comment:
Why Discussion is needed here ?

Reply:
Lol, bad copy/pasta. It was already imported at the top of the function and used. Hence the linter not complaining about it.